python/requirements.txt

```diff
@@ -84,6 +84,7 @@ pytest-rerunfailures
 pytest-sugar
 pytest-lazy-fixture
 pytest-timeout
+pytest-virtualenv
 scikit-learn==0.22.2
 tensorflow==2.5.1
 testfixtures
```

Comment:
@edoakes Add the `virtualenv-clone` to `requirements.txt`?

Reply:
@edoakes `virtualenv-clone` is just a script in only one file, I can copy it to $RAY_HOME/python/ray/_private/runtime_env/clonevirtualenv.py

Reply:
> Can we add the `virtualenv-clone` to `requirements.txt`? @edoakes

I believe you means is add `virtualenv-clone` to ray's requirements in setup.py

Reply:
better if we can copy it instead of adding another dependency

Reply:
cc @pcmoritz virtualenv-clone has an MIT license -- are we ok to pull it into the repo?